tests/system_tests/test_artifacts/test_artifact_cli.py

```diff
@@ -37,13 +37,10 @@ def test_artifact(runner, user):
     assert os.path.exists(path)
 
 
-def test_artifact_put_with_cache_enabled(runner, user, monkeypatch, tmp_path, api):
-    # Use a separate staging directory for the duration of this test.
-    monkeypatch.setenv("WANDB_DATA_DIR", str(tmp_path))
-    staging_dir = Path(get_staging_dir())
-
-    monkeypatch.setenv("WANDB_CACHE_DIR", str(tmp_path))
-    cache = artifact_file_cache.get_artifact_file_cache()
+# Use a separate staging directory for the duration of this test.
```

Comment:
nit: comment isn't necessary

If "override" in the fixture names isn't descriptive enough, you could name them like `temp_env_staging_dir` instead.